packages/manager/apps/hub-next/src/App.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="container-fluid hub-main-view_container">
+    <div id="nav">
+      <router-link to="/">Home</router-link> |
+      <router-link to="/product-details">About</router-link>
```

Comment:
Some products have specific needs due to the way the API is built and need unfortunately some custom route declaration

Reply:
If you're refering to the product-details route yep, the work to get the custom API is done under ProductsList.vue . 
I made this to be able to go to the route when i had static data to be able to test how the route is behaving but this whole section should definetly be deleted later and replaced with a proper breadcrumbs.